Week1/homework/js-exercises/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```

Comment:
This file should not be committed because it belongs to your vscode config. Remove the file from git using `git rm <filename>` and add `.vscode` to your `.gitignore` file so it is not tracked by Git